web/frontend/src/components/slam.vue

```diff
@@ -212,14 +307,42 @@ const selectSLAM2dRefreshFrequency = () => {
 
 const selectSLAMPCDRefreshFrequency = () => {
   updateSLAM3dRefreshFrequency(props.name, selected3dValue);
-};
+}
+
+const refresh3dMap = () => {
+  updateSLAM3dRefreshFrequency(props.name, selected3dValue);
+}
 
 const refresh2dMap = () => {
   updateSLAM2dRefreshFrequency(props.name, selected2dValue);
 };
 
-const refresh3dMap = () => {
-  updateSLAM3dRefreshFrequency(props.name, selected3dValue);
+const handle2dRenderClick = (event: THREE.Vector3) => {
+  updatedDest = true;
+  destinationMarker = event;
+};
+
+const handleUpdateX = (event: CustomEvent<{ value: string }>) => {
```

Comment:
Could this function please be named `handleUpdateDestX` so it is clear which `X` is being updated?